scripts/install/calibrecs.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+if [[ ! -f /install/.calibre.lock ]]; then
+    echo_error "Calibre Content server requires calibre. Please run \`box install calibre\` first."
+    exit 1
+fi
+
+#shellcheck source=sources/functions/utils
+. /etc/swizzin/sources/functions/utils
+
+if [ -z "$CALIBRE_LIBRARY_USER" ]; then
+    if ! CALIBRE_LIBRARY_USER="$(swizdb get calibre/library_user)"; then
+        CALIBRE_LIBRARY_USER=$(_get_master_username)
+        swizdb set "calibre/library_user" "$CALIBRE_LIBRARY_USER"
+    fi
+fi
+
+if [ -z "$CALIBRE_LIBRARY_PATH" ]; then
+    if ! CALIBRE_LIBRARY_PATH="$(swizdb get calibre/library_path)"; then
+        CALIBRE_LIBRARY_PATH="/home/$CALIBRE_LIBRARY_USER/Calibre Library"
+        swizdb set "calibre/library_path" "$CALIBRE_LIBRARY_PATH"
+    fi
+fi
+
+clbServerPath="/home/$CALIBRE_LIBRARY_USER/.config/calibrecs"
+
+_adduser() {
+    ## TODO handle programatically
```

Comment:
What is the status of this todo?

Reply:
We can use this in case we're able to get the compilation for arm going, because the dev added glags for us to do the userdb commands programmatically. the old versions in apt don't support it. Therefore until we have a thing for this, the server is protected under basic auth.